packages/core-components/src/components/calendar/calendar.tsx

```diff
@@ -178,17 +196,25 @@ export class B2bCalendar {
             'b2b-calendar-body': true,
             'b2b-calendar-body--hidden': !this.showCalendar,
           }}>
-          <b2b-calendar-header
-            selectedMonth={this.selectedMonth}
-            selectedYear={this.selectedYear}></b2b-calendar-header>
-          <b2b-calendar-days-header></b2b-calendar-days-header>
-          <b2b-calendar-days
-            selectedMonth={this.selectedMonth}
-            selectedYear={this.selectedYear}
-            selectedDay={this.selectedDay}
-            disableWeekends={this.disableWeekends}
-            disableFutureDates={this.disableFutureDates}
-            disablePastDates={this.disablePastDates}></b2b-calendar-days>
+          {this.showDays && (
+            <div>
```

Comment:
how can i have a containerless wrapper here that groups the components for the daily view, without having to reneder a `<div>`

Reply:
This one was discussed and closed